### ISSUE TITLE
Track changes on Opus cause stream interruption on Chrome

### ISSUE THREAD

**Issue by shoghicp** [Luminarys/kawa#7]
While playing back OGG/Opus streams, a new track will bring the playback to a stop.
To workaround this problem, I have a JS listener on error that tries to restart the stream.

These are the errors that get printed to the console if you log them:

```
PIPELINE_ERROR_DECODE: Failed to send audio packet for decoding: timestamp=93126542 duration=960000 size=19 side_data_size=0 is_key_frame=1 encrypted=0 discard_padding (ms)=(0, 0)
PIPELINE_ERROR_DECODE: Failed to send audio packet for decoding: timestamp=93126542 duration=960000 size=19 side_data_size=0 is_key_frame=1 encrypted=0 discard_padding (ms)=(0, 0)
PIPELINE_ERROR_DECODE: Failed to send audio packet for decoding: timestamp=1028188 duration=960000 size=19 side_data_size=0 is_key_frame=1 encrypted=0 discard_padding (ms)=(0, 0)
```

Using latest kawa, tested in multiple radios

**Comment by shoghicp** [comment]
Seems to be addressed by #9

**Comment by shoghicp** [comment]
This happens again with Opus/FLAC under HTTP/2 mostly and sometimes under HTTP/1.1 too on Firefox
